src/main/java/uk/gov/pay/adminusers/queue/event/EventMessageHandler.java

```diff
@@ -161,9 +161,6 @@ private void handleDisputeLostMessage(Event disputeLostEvent) throws JsonProcess
                 Map<String, String> personalisation = getMinimumRequiredPersonalisation(disputeLostDetails.getGatewayAccountId(),
                         disputeLostEvent.getParentResourceExternalId());
 
-                personalisation.put("disputedAmount", convertPenceToPounds.apply(disputeLostDetails.getAmount()).toString());
```

Comment:
Do we really want to remove these? We will need these in a few weeks time (for PP-9841).

Reply:
Either we remove, or we make sure we only include the fee when it's there otherwise it will break. I opted to remove as it makes sense to add that logic in PP-9841. Not difficult at all to add back in.

Reply:
Guess I could just add in that logic now, as the template is largely defined